fault-tolerance/circuitbreaker/pom.xml

```diff
@@ -11,5 +11,24 @@
     <packaging>war</packaging>
 
     <name>MicroProfile 1.2: Fault Tolerance - CircuitBreaker</name>
-
+    <profiles>
+        <profile>
+            <id>stable</id>
```

Comment:
Should we consider making this default?

Reply:
not really. It's skipping tests that need to be fixed.

Reply:
So when would this profile be used?

Reply:
always on Jenkins. As discussed, could reverse the logic and make a "also-run-unstable-tests" profile or similar.